tests/helpers/test_storage.py

```diff
@@ -91,8 +91,8 @@ async def test_loading_parallel(hass, store, hass_storage, caplog):
 
     results = await asyncio.gather(store.async_load(), store.async_load())
 
-    assert results[0] is MOCK_DATA
-    assert results[1] is MOCK_DATA
+    assert results[0] == MOCK_DATA
```

Comment:
This test needed to be updated because our test mock storage helper abuses pending data to inject mock storage data. Updated the test to verify the data still is the same.